packages/server/src/api/storage/sql/sql.js

```diff
@@ -298,7 +299,7 @@ class SqlStorageMethod {
     const builds = await this._findAll(buildModel, {
       where: {projectId, ...omit(options, ['limit'], {dropUndefined: true})},
       order,
-      limit: options.limit || 10,
+      limit: Number(options.limit) || 10,
```

Comment:
hm this is a bug elsewhere if this isn't `number|undefined` already...

I'll fix that another time, good catch!

Reply:
So the problem was that you can pass only numbers (isNaN is always false) but when you look into the sequelize logs you see that the query looks like
`SELECT * ... LIMIT '100'.`
But that didn't work in mysql it has to be a number like
`SELECT * ... LIMIT 100`
and wir the explicit cast it works correct

Reply:
Oh I know the reason for the fix! :)

Just saying that the typedef expresses `number|undefined` as the type within this function, so this cast shouldn't be *here*. Instead it should be in projects.js where we blindly pass `req.query` which has the limit as the incorrect string type.

Reply:
I will change it